plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -1377,7 +1377,12 @@ $.extend(DataTable.prototype, UIControl.prototype, {
 
     getFilterParams: function (params) {
         return Object.keys(params)
-           .filter(key => key.startsWith('filter_column') || key.startsWith('filter_pattern'))
+           .filter(key => {
+               return key === 'filter_trigger_id'
```

Comment:
does it make sense to store that when there are multiple tables that had a search? It seems it would then highlight the last table with a search - and not the table I used last to search.

Reply:
Yeah, it looks like ideally we would store the latest focused field across all the application, not per report, since there can only be on. And also remove it from the storage on blur so that we don't refocus a field that shouldn't be active, i.e. when you click out from the search before changing dates/segments.

Reply:
Technically you will always have a `blur` event on the search input when you change the date, as you need to click somewhere in the period chooser. 
Personally I would not even expect the search input to be focus automatically in that case. But I'm not a UX specialist, so be better to ask Shelby maybe 🤔

Reply:
Removed both `filter_sort` and `filter_trigger_id` from the list